ddtrace/internal/utils/wrappers.py

```diff
@@ -20,4 +20,4 @@ def iswrapped(obj, attr=None):
 def unwrap(obj, attr):
     # type: (Any, str) -> None
     f = getattr(obj, attr)
-    setattr(obj, attr, f.__wrapped__)
+    setattr(obj, attr, getattr(f, "__wrapped__", f))
```

Comment:
why this change? I didn't see any calls to `unwrap` in this diff.

also, should whoever is calling `unwrap` just do `if iswrapped(obj, attr): unwrap(obj, attr)` ?

Reply:
(I am not opposed to this change, but also looking like `iswrapped` also looks for `__dd_wrapped__`, maybe we need to account for that here? or maybe we should try and migrate whoever is doing this wrapping to using the bytecode wrapping `ddtrace.internal.wrapping.wrap/unwrap` instead)

Reply:
Reverting this change.